charts/gha-runner-scale-set/templates/_helpers.tpl

```diff
@@ -102,6 +102,8 @@ args:
   - --host=unix:///var/run/docker.sock
   - --group=$(DOCKER_GROUP_GID)
 env:
+  - name: RUNNER_WAIT_FOR_DOCKER_IN_SECONDS
```

Comment:
Do we want to default to a lower value or is this max min duration we need to fix the issue? I can already see the issues a mile away for `why is it taking longer to...`